src/ProgressBar.js

```diff
@@ -2,20 +2,14 @@ import classNames from 'classnames';
 import React, { cloneElement } from 'react';
 import PropTypes from 'prop-types';
 
-import {
-  bsClass as setBsClass,
-  bsStyles,
-  getClassSet,
-  prefix,
-  splitBsProps
-} from './utils/bootstrapUtils';
-import { State } from './utils/StyleConfig';
+import { createBootstrapComponent } from './ThemeProvider';
+
 import ValidComponentChildren from './utils/ValidComponentChildren';
 
 const ROUND_PRECISION = 1000;
 
 /**
- * Validate that children, if any, are instances of `<ProgressBar>`.
+ * Validate that children, if any, are instances of `Bootstrap(<ProgressBar>)`.
```

Comment:
We can leave this as `ProgressBar` as user won't have a concept of what the wrapper component is here since it's an implementation detail.

Reply:
Lets also explicitly add `bsPrefix` and `variant` to the prop-types with docs comments.  For the bsPrefix if can be very simple like: https://github.com/react-bootstrap/react-bootstrap/blob/v4-support/src/ButtonGroup.js#L13